docs/images/templates/update.png:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]

```

Comment:
what is this file?

Reply:
Good catch. Thanks!
(It's an artifact from Windows WSL. I'll delete it.)